ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -212,6 +214,10 @@ Item {
                 const contactDetails = publicKey === "" ? {} : Utils.getContactDetailsAsJson(publicKey, true, true)
                 Global.blockContactRequested(publicKey, contactDetails)
             }
+            onRemoveFromGroup: (publicKey) => {
+                const chatId = root.store.chatCommunitySectionModule.activeItem.id
+                root.store.chatCommunitySectionModule.removeMemberFromGroupChat("", chatId, publicKey)
```

Comment:
Just a suggestion, but we could create a `removeMemberFromActiveGroupChat(publicKey)` function in the store that would do the two calls above and hide some of the logic. No need to use `chatCommunitySectionModule` in the component that way.

Reply:
Yes, that's also important rule (affects testing) that context properties should be accessed only from stores and not exposed from them directly. So `chatCommunitySectionModule` should not be used (and even accessible) from `root.store` directly like `root.store.chatCommunitySectionModule`.